specifications/api/swagger.yaml

```diff
@@ -5199,3 +5231,40 @@ paths:
           application/json:
             schema: *ApiSharedWalletPatchData
       responses: *responsesPatchSharedWallet
+
+  /shared-wallets/{walletId}/keys/{role}/{index}?hash=false:
```

Comment:
I am not sure if query params should be exposed like this.Nevertheless, 
```
openapi-spec-validator --schema 3.0.0 specifications/api/swagger.yaml
```
passes

Reply:
I fixed it for you.